src/test/assembly/is_aligned.rs

```diff
@@ -13,7 +13,7 @@
 // CHECK: decq
 // CHECK-NEXT: testq
 // CHECK-NEXT: sete
-// CHECK-NEXT: retq
+// CHECK: retq
```

Comment:
are there other statements in between that we could check with `CHECK-NEXT`?

Reply:
instead of just switching to `CHECK`, can we add more `CHECK-NEXT` before this line and keep it at `CHECK-NEXT` or does it vary what is in between?

Reply:
On apple there's a `popq %rbp` before the `retq`, but not on linux or windows. I haven't found a way do an optional `CHECK-NEXT` yet.

Maybe we can use [`-C force-frame-pointers`](https://doc.rust-lang.org/rustc/codegen-options/index.html#force-frame-pointers) (on or off) instead?

Reply:
Thanks! that's kinda what I expected, but wanted to be sure